tests/regression/roles/sonic_poe/defaults/main.yml

```
@@ -0,0 +1,125 @@
+---
+ansible_connection: httpapi
+module_name: sonic_poe
+
+tests:
+  - name: test_case_01_no_card_support
+    description: test merging all config
```

```suggestion
    description: test merging of config
```

(The word "all" is confusing here.)